integration-tests/scripts/install.sh

```diff
@@ -53,18 +54,17 @@ github_integration() {
     GITOPS__GIT_TOKEN="${GITOPS__GIT_TOKEN:-$(cat /usr/local/rhtap-cli-install/github_token)}"
     GITHUB__APP__WEBHOOK__SECRET="${GITHUB__APP__WEBHOOK__SECRET:-$(cat /usr/local/rhtap-cli-install/rhdh-github-webhook-secret)}"
 
-    cat <<EOF >>"$tpl_file"
-integrations:
-  github:
-    id: "${GITHUB__APP__ID}"
-    clientId: "${GITHUB__APP__CLIENT__ID}"
-    clientSecret: "${GITHUB__APP__CLIENT__SECRET}"
-    host: "github.com"
-    publicKey: |-
-$(printf "%s\n" "${GITHUB__APP__PRIVATE_KEY}" | sed 's/^/      /')
-    token: "${GITOPS__GIT_TOKEN}"
-    webhookSecret: "${GITHUB__APP__WEBHOOK__SECRET}"
-EOF
+    sed -i "/integrations:/ a \  github:\n\
```

Comment:
I feel like this would be much more readable if we used a tool like `yq` instead of `sed`.